File: examples/db/delete/node/deleteARecord.js
const { DbService } = require("micro-js-client/db");

// Delete a record in the database by id.
async function deleteArecord() {
  let dbService = new DbService(process.env.MICRO_API_TOKEN);
  let rsp = await dbService.delete({
    id: "1",
    table: "users",
  });
  console.log(rsp);
}

deleteArecord();

File: examples/db/delete/go/deleteARecord.go
package example

import (
	"fmt"
	"github.com/micro/services/clients/go/db"
	"os"
)

// Delete a record in the database by id.
func DeleteArecord() {
	dbService := db.NewDbService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := dbService.Delete(&db.DeleteRequest{
		Id:    "1",
		Table: "users",
	})
	fmt.Println(rsp, err)
}
